skyve-war/src/main/java/modules/admin/Tag/TagService.java

```diff
@@ -0,0 +1,144 @@
+package modules.admin.Tag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.skyve.CORE;
+import org.skyve.EXT;
+import org.skyve.domain.Bean;
+import org.skyve.metadata.customer.Customer;
+import org.skyve.metadata.model.document.Document;
+import org.skyve.metadata.module.Module;
+import org.skyve.metadata.user.User;
+import org.skyve.persistence.AutoClosingIterable;
+import org.skyve.persistence.Persistence;
+import org.skyve.persistence.SQL;
+import org.skyve.tag.TagManager;
+
+import jakarta.enterprise.inject.Default;
+
+/**
+ * This class acts as a service layer to encapsulate domain logic.
+ *
+ * Add this line to classes that wish to use it: @Inject private transient TagService tagService;
+ */
+@Default
+public class TagService {
+	/**
+	 * Add to the subject tag records from the object tag.
+	 * 
+	 * @param subject
+	 * @param object
```

Comment:
Clean up Javadoc when we touch it. Parameters should have descriptions for developers using the service so they know what is expected.